go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	cosmossdk.io/math v1.0.0-beta.3
 	github.com/BurntSushi/toml v1.2.1
 	github.com/alecthomas/jsonschema v0.0.0-20200530073317-71f438968921
-	github.com/celestiaorg/celestia-app v0.10.0-rc1
+	github.com/celestiaorg/celestia-app v0.10.0
```

Comment:
Are we releasing with 1.8.0-0.34.20 core? I don't understand why this PR is closing this issue https://github.com/celestiaorg/celestia-node/issues/1384

And we are still in v.1.5.0-0.34.20 at this release PR

Reply:
Sorry, we should have created a patch release which includes that change so we could have included in this one. 

It will definitely be in the next release

Reply:
Good catch @Bidon15. We can still close the issue as it's about updating the app/core and not about fixed race, we just need to change the issue description not to include race fix

Reply:
> we just need to change the issue description not to include race fix

just updated the description with the commit of the only additional change that might help CI as to differentiate between the other one in core

Reply:
IDK, @evan-forbes ?